docs/boot-tutorial.md

```diff
@@ -267,7 +267,7 @@ linux_binaries = {
                 cp ../linux-configs/config.{version} .config;
                 make -j8;
                 cp vmlinux vmlinux-{version};
-                '''.format(v='5.2.3'),
+                ''',
                 inputs = [experiments_repo, linux_repo,],
                 documentation = f"Kernel binary for {version} with simple "
```

Comment:
I think this needs a `.format`

Reply:
Is .format really needed? These are f strings and {version} is defined by the outer loop (which is iterating through a list of different linux versions)